pkgs/code_assets/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## 0.19.4-wip
+## 0.19.4
```

Comment:
You previous PR changed this from `0.19.5-wip` to `0.19.4-wip`, this should be `0.19.5`